src/components/modals/AutoLogoutModal.tsx

```diff
@@ -67,92 +63,88 @@ export const AutoLogoutModal = (props: Props) => {
     bridge.resolve(undefined)
   })
 
-  // Default to `hours` if it's disabled:
+  // Default to hours if it's disabled:
   React.useEffect(() => {
     if (displayTime.value === 0) setDisplayTime({ ...displayTime, measurement: 'hours' })
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
+  const initPositionNumber = React.useMemo(
```

Comment:
You can use `const [initPositionNumber] = React.useState(displayTime.value)` to do the same thing without ESLint cheats. We'll capture the initial value into the state, and then just never update it.